src/components/hide-this-page/default/index.njk

```diff
@@ -0,0 +1,27 @@
+---
+title: Hide this Page
+layout: layout-example.njk
+---
+
+<!-- Code sits above the 'skip to main content' <div> to warn users that this feature exists on the page -->
+<div class="govuk-quick-exit__container govuk-visually-hidden">
+  <p class="govuk-body">
+    To leave the page quickly, press the escape key.
+  </p>
+</div>
+
+<!-- Code used for the component and sits inside the <main> content wrapper -->
+<div class="govuk-quick-exit__container govuk-quick-exit__enabled">
+  <div class="govuk-quick-exit__position">
+    <a href="https://www.google.com" id="quick-exit" tabindex="-1" class="govuk-button govuk-button--warning govuk-quick-exit">
+      <strong>Hide this page</strong>
+      <span class="govuk-visually-hidden">Or press escape key to hide this page</span>
+    </a>
+    <p class="govuk-body govuk-quick-exit--info govuk-!-font-size-14">
+      (Or press Esc key)
+    </p>
```

Comment:
I think that the "(Or press Esc key)" inside the `govuk-quick-exit--info` container and the visually hidden "Or press escape key to hide this page" shouldn't be there when JavaScript isn't available. 

They still appear to be both readable and visible when JavaScript is turned off, which means that pressing the escape key doesn't do anything.